web/auth.html

```diff
@@ -3,19 +3,15 @@
 <p>Authentication is complete. If this does not happen automatically, please close the window.</p>
```

Comment:
IMO, the content of this should be change too. if this page can not redirect, we should have the link to user click to go back to `redirectUrl + '/?loginToken=' + loginToken;`. WDYT?

Reply:
Good idea. i have updated the link 

https://github.com/linagora/twake-on-matrix/assets/43041967/5a9354bf-64c6-40a4-95e1-f519b37d8673